# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/apz/util/ContentProcessController.cpp
# Commit: 4230bdcf07ae
# Full Hash: 4230bdcf07ae69e03a9e241f10c201c38e6ca995
# Author: Mike Conley <mconley@mozilla.com>
# Date: 2022-06-23 03:54:50
# Regressor Bug: 1773865
# File Overlap Count: 1
# Description:
#   Bug 1773865 - Dispatch an event on the window document when a pinch zoom gesture ends. r=botond,smaug,NeilDeakin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D149283
# ==============================================================================

diff -r d1ddb1ce916b -r 4230bdcf07ae gfx/layers/apz/util/ContentProcessController.cpp
--- a/gfx/layers/apz/util/ContentProcessController.cpp	Wed Jun 22 16:08:22 2022 +0000
+++ b/gfx/layers/apz/util/ContentProcessController.cpp	Wed Jun 22 16:15:57 2022 +0000
@@ -97,6 +97,12 @@
   MOZ_ASSERT_UNREACHABLE("Unexpected message to content process");
 }
 
+void ContentProcessController::NotifyScaleGestureComplete(
+    const ScrollableLayerGuid& aGuid, float aScale) {
+  // This should never get called
+  MOZ_ASSERT_UNREACHABLE("Unexpected message to content process");
+}
+
 bool ContentProcessController::IsRepaintThread() { return NS_IsMainThread(); }
 
 void ContentProcessController::DispatchToRepaintThread(